index.html

```diff
@@ -131,6 +133,72 @@ <h1 id="placeholder-img-head">Profesional Art Printing Data</h1>
 </button>
 </div>
 </div>
+</section>
+<section id="desktop-card">
+    <div id="desktop-card-1">
+        <h1>
+        </h1>
+        <p></p>
+        
```

Comment:
- [ ] [OPTIONAL] This `<p>` and `<h1>`  look empty. I don't know the reason why you left it empty but it would be better if you remove it 😃 👍

Reply:
i have been working on my desktop version next so this empty h1 and p tags realtes to desktop but still I remove that